docs/tutorials/intro_tutorial.rst

```diff
@@ -24,7 +24,7 @@ model as it runs. Finally, we go over how to write your own
 visualization module, for users who are comfortable with JavaScript.
 
 You can also find all the code this tutorial describes in the
-**examples/Tutorial-Boltzmann\_Wealth\_Model** directory of the Mesa
+`examples/boltzmann_wealth_model/ <https://github.com/projectmesa/mesa/tree/master/examples/boltzmann_wealth_model>`__ directory of the Mesa
 repository.
 
 Sample Model Description
```

Comment:
(Not Part of the PR, but still:)
The literature references (line 35 and line 162 are still broken)

Reply:
After the intro tutorial rewrite in the `doc_builds` branch, this link no longer exists.